azurerm/resource_arm_container_group.go

```diff
@@ -599,18 +622,33 @@ func expandContainerGroupContainers(d *schema.ResourceData) (*[]containerinstanc
 	return &containers, &containerGroupPorts, &containerGroupVolumes
 }
 
-func expandContainerEnvironmentVariables(input interface{}) *[]containerinstance.EnvironmentVariable {
+func expandContainerEnvironmentVariables(input interface{}, secure bool) *[]containerinstance.EnvironmentVariable {
+
 	envVars := input.(map[string]interface{})
 	output := make([]containerinstance.EnvironmentVariable, 0)
```

Comment:
output := make([]containerinstance.EnvironmentVariable, 0, len(envVars)) might be helpful to reduce reallocation. Depends on you, it might not be so helpful coz the size of variables will be limited.